python/pyspark/ml/tests.py

```diff
@@ -1027,6 +1027,21 @@ def test_storage_levels(self):
         self.assertEqual(als.getFinalStorageLevel(), "DISK_ONLY")
         self.assertEqual(als._java_obj.getFinalStorageLevel(), "DISK_ONLY")
 
+    def test_unknown_strategy(self):
```

Comment:
@BryanCutler yeah, it would fail when `fit` is called - on the JVM side actually as that is where the validation occurs. It's due to the wrapper as you say, not much we can do on the Python side for now.

Since we test invalid params in the Scala suite, I don't think it's necessary to test them again here.